profiles/upload/h3n2/ha/prioritized_seqs_file.tsv

```diff
@@ -0,0 +1,4 @@
+strain	accession
+A/Croatia/10136RV/2023-egg	EPI3356221
```

Comment:
Looking back at the table of different Croatia isolates, I see that all of the cell-passaged isolates have the same haplotype which is why I didn't mention a specific accession to prefer for that strain. I can imagine, though, that someone uploads another cell-passaged isolate of that strain in the future with a slightly different haplotype from the passaging and we unknowingly switch to that new version in our analyses.

Could we future-proof ourselves a bit by specifying the cell-passaged accession for Croatia, too? I would pick the isolate with the fewest passages which is EPI_ISL_19085723 (HA accession: EPI3250718 and NA accession: EPI3250717).

Reply:
Added in a1ec64d